src/template/index.js

```diff
@@ -267,6 +268,7 @@ async function google(req, res) {
 
     const [subdomain] = req.headers.host.split('.');
     const [country, region, ...servicename] = subdomain.split('-');
+    const [packageName, name] = process.env.K_SERVICE.split('--');
```

Comment:
not sure about this. @trieloff  can you give an example of the `K_SERVICE` ?

Reply:
`simple-package--simple-name`